src/components/Popup/IQServer/SecurityPage/SecurityPage.tsx

```diff
@@ -49,11 +49,18 @@ const SecurityPage = (props: SecurityProps): JSX.Element | null => {
     ) {
       const purl = nexusContext.policyDetails.results[0].component.packageUrl;
       const securityData = nexusContext.policyDetails.results[0].securityData;
-
+      const sortedIssues = securityData.securityIssues
+        .sort((a: SecurityIssue, b: SecurityIssue) => {
+          return b.reference > a.reference ? 1 : -1;
+        })
+        .sort((a: SecurityIssue, b: SecurityIssue) => {
+          return b.severity - a.severity;
+        });
+      console.log('sortedIssues', sortedIssues);
```

Comment:
Switch to something like `console.trace` or `console.info` for stuff like this, that way it's easier to tune down the logs!